src/model/album.rs

```diff
@@ -67,22 +62,117 @@ pub struct FullAlbum {
 }
 
 /// Full Albums
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct FullAlbums {
     pub albums: Vec<FullAlbum>,
 }
 
 /// [link to get list new releases](https://developer.spotify.com/web-api/get-list-new-releases/)
 /// Simplified Albums wrapped by Page object
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct PageSimpliedAlbums {
     pub albums: Page<SimplifiedAlbum>,
 }
 
-/// [link to save album object](https://developer.spotify.com/web-api/object-model/#save-album-object)
+/// [link to save album object](https://developer.spotify.com/documentation/web-api/reference/object-model/#save-album-object)
 /// Saved Album object
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 pub struct SavedAlbum {
     pub added_at: DateTime<Utc>,
     pub album: FullAlbum,
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::model::CopyrightType;
+    #[test]
+    fn test_full_album() {
+        let json_str = r#"
```

Comment:
>  Can't they be reduced to what we actually want to test?

If we want to the serialization and deserialization of some struct, it's no way to reduce the `json_str`, since it's necessary to deserialize a struct.

Perhaps we could move all model tests into a single file, instead of spreading over to different files.

Reply:
I think that as long as we don't include one of these strings for *every single endpoint*, we're fine. We can just re-use it for several tests.

Reply:
Yep, it makes sense. I would try to reduce duplicated json string and re-use them.